src/VerifyPIVController.js

```diff
@@ -0,0 +1,131 @@
+/*!
+ * Copyright (c) 2019, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+/* eslint complexity:[2, 10], max-params: [2, 11] */
+
+define([
+  'okta',
+  'util/FormController',
+  'util/FormType',
+  'views/shared/FooterWithBackLink'
+],
+function (Okta, FormController, FormType, FooterWithBackLink) {
+
+  var _ = Okta._,
+      $ = Okta.$;
+  var { Util } = Okta.internal.util;
+
+  return FormController.extend({
+    className: 'piv-cac-card',
+    Model: {
+
+      save: async function () {
+        this.trigger('request');
+        var self = this,
+            data = {
+              fromURI: this.settings.get('relayState')
+            },
+            pivConfig = this.settings.get('piv');
+
+        try {
+          await this.getCert(pivConfig.certAuthUrl);
+          const res = await this.postCert(pivConfig.certAuthUrl, data);
+          Util.redirect(res.redirectUrl);
+        } catch (err) {
+          self.trigger('error', self, err);
+        }
+      },
+
+      getCert: function (certAuthUrl) {
+        return $.get({
```

Comment:
nit:
instead of using jquery, maybe we could try to use `cross-fetch` which is used by auth-js underneath? (separated PR if you have time)

Reply:
Spoke to Haisheng offline. Will need to move httpClient.js from v2 src to shared. Also will need to investigate behavior on different browsers (probably provide capability to override / exclude some headers). Will do this as a separate PR in Dec.